packages/page-objects/src/components/editor/ExtensionEditorView.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License", destination); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Editor, WebView } from '../..';
+
+export class ExtensionEditorView extends Editor {
+	constructor() {
+		super(undefined, ExtensionEditorView.locators.ExtensionEditorView.constructor);
+	}
+
+	getWebView(): WebView {
```

Comment:
I am not sure this method suits to API of this page object? the `getReadme` will be implemented as part of `ExtensionEditorDetailsSection ` when it will be unblocked?